src/styles/components/wizard.less

```diff
@@ -24,7 +24,7 @@
 .wizard-progress-container {
   height: 2em;
   margin: 3.5em 0 2.5em;
-  background-image: url(../../images/dot.png);
+  background-image: url(../images/dot.png);
```

Comment:
so this was just wrong before? ouch!

Reply:
Not sure, because it was working before. Perhaps the path was more forgiving before.